addons/website_forum/views/website_forum.xml

```diff
@@ -807,7 +810,12 @@
         </div>
         <div class="question-block">
             <div class="pull-right author-box">
-                <img class="pull-left img img-rounded img-avatar" t-attf-src="/forum/user/#{answer.create_uid.id}/avatar"/>
+                <div class="o_forum_user_bio_expand" t-if="answer.can_display_biography">
```

Comment:
@tde-banana-odoo We will be having access error (while getting answer.create_uid.karma) for public user if we check the condition directly here. I added the computed field because the method is accessing 'forum.post' as sudo.